pkg/cmd/container/create.go

```diff
@@ -357,8 +365,49 @@ func generateRootfsOpts(args []string, id string, ensured *imgutil.EnsuredImage,
 			// error message is from Podman
 			return nil, nil, errors.New("no command or entrypoint provided, and no CMD or ENTRYPOINT from image")
 		}
+
+		entrypointPath = processArgs[0]
+
 		opts = append(opts, oci.WithProcessArgs(processArgs...))
 	}
+
+	systemdPaths := []string{
+		"/usr/sbin/init",
+		"/sbin/init",
+		"/usr/local/sbin/init",
```

Comment:
Is there any image that has `/usr/local/sbin/init`?

Reply:
I set these to be the same as `podman`:
```true enables systemd mode only when the command executed inside the container is systemd, /usr/sbin/init, /sbin/init or /usr/local/sbin/init.
```

(But, most images I have seen use `/sbin/init`)

https://docs.podman.io/en/latest/markdown/podman-run.1.html#systemd-true-false-always